src/containers/onboarding/index.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { useEffect, useState } from 'react';
+import { Button, Card, Form, Input, Typography } from 'antd';
+import styled from 'styled-components';
+import { ResponseCard } from '../../components/onboarding';
+import ProtectedApiClient from '../../api/protectedApiClient';
+import { OnboardingResponseData } from '../../api/protectedApiClient';
+import {
+  AsyncRequest,
+  AsyncRequestCompleted,
+  AsyncRequestFailed,
+  asyncRequestIsComplete,
+  AsyncRequestLoading,
+  AsyncRequestNotStarted,
+} from '../../utils/asyncRequest';
+
+const { Title, Paragraph } = Typography;
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+  width: 100%;
+`;
+
+const FormCard = styled(Card)`
+  display: flex;
+  width: 50%;
+  border-radius: 5px;
+`;
+
+const OnboardingPageTitle = styled(Title)`
+  display: flex;
+  width: 100%;
+  justify-content: center;
+`;
+
+const FormInput = styled(Input)`
+  display: flex;
+  width: 80%;
+  border: 1px 1px black;
+`;
+
+const StyledButton = styled(Button)`
+  width: 30%;
+  margin-top: 16px;
+`;
+
+const ResponseContainer = styled.div`
+  width: 75%;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  margin-top: 16px;
+`;
+
+const SuccessMessage = styled(Paragraph)`
+  color: green;
+`;
+
+interface OnboardingRequestData {
```

Comment:
Duplicated type between here and the api client? choose one and export it or make another file